calibrate.py

```diff
@@ -27,19 +27,28 @@ def acquire_files():
             elif filename == "UCB018_Soil_Sample010_2.Spe":
                 pass
             else:
-                if '_recal' in filename:
+                if '_recal.Spe' in filename:
```

Comment:
Use `filename.lower()` and `'_recal.spe'` rather than assuming the case of the extension.